common/protob/messages-common.proto

```diff
@@ -50,7 +50,8 @@ message Failure {
  */
 message ButtonRequest {
     optional ButtonRequestType code = 1;  // enum identifier of the screen
-    optional uint32 pages = 2;            // if the screen is paginated, number of pages
+    optional string _data = 2 [deprecated=true];
```

Comment:
I don't believe `data` was ever used for anything at all. Do you have usages in codebase that say something else?

Reply:
Trezor connect does not appear to touch it directly but it simply [forwards any button request](https://github.com/trezor/connect/blob/1ea2accd177eaa0c2c2c9348b47b515161ae3193/src/js/device/DeviceCommands.js#L406) further to clients. So if it was ever used, it might still be somewhere out in the wild. So what is the optimal solution? keep the field there as deprecated? or make it reserved?

Reply:
but this is [a recent change](https://github.com/trezor/connect/commit/2faa0bcf5c639ed6740eb760381d896d758c0331), in particular, _after_ this field was already `pages`.

If you don't have any usages of `_data` in code, I'd prefer to leave this as-is.